package.json

```diff
@@ -54,7 +54,7 @@
     "tap-spec": "4.1.1",
     "tape": "4.8.0",
     "tape-watch": "2.3.0",
-    "typescript": "2.7.2",
+    "typescript": "^4.1.3",
```

Comment:
Rolled up TS to use the type generics on [`type` declarations](https://github.com/danpaz/bodybuilder/pull/283/files#diff-2c7bfb4dd3673d112a912c3f9320fb07eb59e3e54317dc5d2d147d187088ae18R358), which I guess wasn't available in 2.7. Let me know if there's any issue